lib/libutils/isoc/bget.c

```diff
@@ -1028,10 +1034,10 @@ void brel(buf, poolset, wipe)
 	   memory.  */
 
 	bn = BFH(((char *) b) + b->bh.bsize);
+        wipe_size = wipe_size + sizeof(struct bfhead);
     }
     if (wipe) {
-	V memset_unchecked(((char *) b) + sizeof(struct bfhead), 0x55,
-			   (MemSize) (b->bh.bsize - sizeof(struct bfhead)));
+	V memset_unchecked(wipe_start, 0x55, (MemSize)wipe_size);
```

Comment:
The cast isn't needed `wipe_size` already has this type.

Reply:
I've removed this cast.